easyInterface/Diffraction/Interface.py

```diff
@@ -352,7 +368,7 @@ def updatePhases(self) -> NoReturn:
             self.project_dict.setItemByPath(['info', 'phase_ids'], list(phases.keys()))
             self.project_dict.endBulkUpdate()
         else:
-            k, v = self.project_dict['phases'].dictComparison(phases)
+            k, v, t = self.project_dict['phases'].dictComparison(phases)
```

Comment:
Should we use a less cryptic names, such as `keys`, `values`? And what is `t`? It seems that `t` is not used.

Reply:
Modified